tests/manage/ocs_288.py

```diff
@@ -0,0 +1,407 @@
+"""
+A test for creating pvc with random sc
+"""
+import base64
+import logging
+import os
+import random
+import string
+
+import pytest
+import yaml
+
+from ocs import defaults
+from ocs import ocp
+from ocsci import ManageTest, tier1
+from utility import templating
+
+log = logging.getLogger(__name__)
+CEPH_FILESYSTEM = os.path.join("templates/ocs-deployment",
+                               "cephfilesystem.yaml")
+SC_RBD_YAML = os.path.join("templates/CSI/rbd",
+                           "storageclass.yaml")
+SECRET_RBD = os.path.join("templates/CSI/rbd", "secret.yaml")
+SC_CEPHFS_YAML = os.path.join("templates/CSI/cephfs",
+                              "storageclass.yaml")
+RBD_POOL_YAML = os.path.join("templates/CSI/rbd", "CephBlockPool.yaml")
+PVC_RBD_YAML = os.path.join("templates/CSI/rbd", "pvc-rbd.yaml")
+SECRET_CEPHFS = os.path.join("templates/CSI/cephfs",
+                             "secret.yaml")
+TEMP_YAML_FILE = 'temp.yaml'
+TEMP_YAML_FILE_FS = 'temp_fs.yaml'
+
+SC = ocp.OCP(
+    kind='StorageClass', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+POOL = ocp.OCP(
+    kind='CephBlockPool', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+POD = ocp.OCP(
+    kind='Pod', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+SECRET = ocp.OCP(
+    kind='Secret', namespace="default"
+)
+PVC = ocp.OCP(
+    kind='PersistentVolumeClaim', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+CFS = ocp.OCP(
+    kind='CephFilesystem', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+storageclass_list = []
+pvc_list = []
+secret_list = []
+
+
+@pytest.fixture(scope='class')
+def test_fixture(request):
+    """
+    Create disks
+    """
+    self = request.node.cls
+
+    def finalizer():
+        teardown(self)
+
+    request.addfinalizer(finalizer)
+    setup(self)
+
+
+def setup(self):
+    """
+    Setting up the environment for the test
+    """
+    assert create_secret(file_name=SECRET_RBD)
+    assert create_secret(file_name=SECRET_CEPHFS)
+    assert create_cephfilesystem()
+    assert create_storageclass_rbd(get_pool_get_sc_name())
+    assert create_storageclass_rbd(get_pool_get_sc_name())
+    assert create_storageclass_rbd(get_pool_get_sc_name())
+    assert create_storageclass_rbd(get_pool_get_sc_name())
+    assert create_storageclass_rbd(get_pool_get_sc_name())
+    assert create_storageclass_cephfs()
+
+
+def teardown(self):
+    """
+    Tearing down the environment
+    """
+    assert delete_pvc(pvc_list)
+    assert delete_storageclass_and_cephblockpool(storageclass_list)
+    assert delete_cephfilesystem(file_name=TEMP_YAML_FILE_FS)
+    assert delete_secret(secret_list)
+
+
+def delete_pvc(pvc_list):
+    """
+    Function to delete pvc
```

Comment:
fix docstring according to coding guidelines:
https://github.com/red-hat-storage/ocs-ci/blob/master/docs/coding_guidelines.md